roles/logs/tasks/logs_to_hdd.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Create probably missing log folders on external HDD
+  file:
+    path: "{{ bsf_log_folder }}/log/{{ item.name }}"
+    owner: "{{ item.owner }}"
+    group: "{{ item.group }}"
+    mode: 0775
```

Comment:
0755 for these directories. `/var/log/bsf/` being an exception (which I don't like but that's another topic).

Reply:
What do you recommend then ?

Reply:
Currently writable by `adm` group. Only `telegraf` in this group (even `ideascube` is not, despite being suder :p). I am not fond of adding system users to the `adm` group.

The common practice is to create a dedicated group, see `/etc/group` for a tons of examples.

Reply:
Let me put some emphasis about this:

My remark is beyond the scope of this pull request, and pull requests should be as atomic as commits should be. Let's keep it like that in here, and fix the group/ownership later.

Reply:
Ok, let's do that